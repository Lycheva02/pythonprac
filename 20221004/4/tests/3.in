"642/x", "(22 - x)**3", "x**2 - y**2" 
6