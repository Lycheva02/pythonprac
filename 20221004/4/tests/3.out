-16765767.0
